.github/workflows/release.yaml

```diff
@@ -333,3 +333,42 @@ jobs:
       base64-subjects: "${{ needs.combine_hashes.outputs.hashes }}"
       upload-assets: true # Optional: Upload to a new release
       provenance-name: kargo-cli.intoto.jsonl
+
+  update-homebrew-formula:
+    if: github.event_name == 'release'
```

Comment:
I think there are additional conditions...

I think we would want to exclude RCs and patch releases _not_ cut from the latest release-1.x branch.

idk how to do this off the top of my head.

You might be able to borrow bits of the logic used by the nightly to determine what the latest stable release is.

Reply:
I thought about that as well. I'm open to that over the short-term.

I'm going to test this on my own Kargo fork and private tap in a bit.